cypress/e2e/urlHandling.cy.ts

```diff
@@ -30,6 +30,22 @@ describe('URL handling', () => {
 
       cy.title().should('eq', 'Test | Playroom');
     });
+
+    it('editor hidden', () => {
+      cy.visit(
+        'http://localhost:9000/#?code=N4IgpgJglgLg9gJwBJQhMA7EAuGCCuYAvkA'
+      );
+
+      cy.get('textarea').should('not.be.focused');
+
+      // Todo - write a test that checks the CodeMirror element is not visible
```

Comment:
Feel like this todo is redundant given the explanation below.

Reply:
I think I'll keep the todo only but remove the surrounding comment, so when a fix is found it prompts us to change it for both tests